### ISSUE TITLE
Add RSS Subscribe button at top of each group listing

### ISSUE THREAD

**Issue by ntrel** [CyberShadow/DFeed#154]
I didn't realize you can get an RSS feed of new posts in a group. Although at the moment the feed is empty, maybe because (posting to) the forum is not working ATM?

It would be good to add a _Subscribe_ button next to the _Create thread_ button that sets the group field to that group and checks new posts only.

**Comment by CyberShadow** [comment]
> Although at the moment the feed is empty, maybe because (posting to) the forum is not working ATM?

Yeah. I think the feed URL has a parameter for how many days to fetch.

> It would be good to add a _Subscribe_ button next to the _Create thread_ button that sets the group field to that group and checks new posts only.

When web browsers had built-in RSS readers, they used to show a small icon in the address bar or elsewhere in the UI that allowed subscribing to the feeds advertised on the page. This was true when the feature was written.

I don't think we can discover if users have a working RSS/ATOM reader set up, so I'm not sure about showing such a button by default. However, today I think it would make more sense to add support for push notifications. They are universally supported, and shouldn't be too difficult to implement. There is already a subscription system with multiple mechanisms, including an ATOM feed, so this would be one more mechanism.
